index.html

```diff
@@ -30,10 +30,44 @@
             <li class="toggle-ul-li"><a class="toggle-ul-li-a" href="#form-section">Contact</a></li>
         </ul>
     </div>
+    <div class="popup-window">
+        <div class="popup-window-content">
+            <div class="top-content">
+                <div class="top-content-text flex">
+                    <h2 id="firsth2">Keeping track of hundreds of components website</h2>
+                    <h2 id="secondh2">Multi post stories</h2>
+                    <i class="fa-regular fa-x"></i>
+                </div>
+                <ul class="top-content-tag flex">
+                    <li class="html-tag">html</li>
+                    <li class="bootstrap-tag">Bootstrap</li>
+                    <li class="ruby-tag">Ruby on rails</li>
+                </ul>
+            </div>
+            <div class="popup-content">
+                <img id="snapshot-portfolio" src="./images/Snapshoot Portfolio.svg" alt="Snapshoot Portfolio">
+                <div class="popup-content-text flex">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. This has
+                        been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                        standard dummy text</p>
+
+                    <div class="popup-content-text-btn flex">
+                        <button type="button" class="pop-btn flex"><a
+                                href="https://torobucci.github.io/Portfolio-finish-mobile-version/">See live</a><img
+                                src="./images-desktop/Icon-see live.svg" alt=""></button>
+                        <button type="button" class="pop-btn"><a
+                                href="https://github.com/torobucci/Portfolio-finish-mobile-version">See source</a><img
+                                src="./images-desktop/Icon-see-source.svg" alt=""></button>
 
+                    </div>
+                </div>
+
+            </div>
+        </div>
+    </div>
```

Comment:
#### OPTIONAL

- [ ] Based on the requirements, `Modal popup data is populated with the data from the projects array` this means that for each project card button  a user clicks, the content of the modal popup should be the same with the content for the particular project card, currently, you are displaying the same modal popup for all cards please change it, when I click the popup button it should show me more information pertaining to that project card I clicked, please use javascript to achieve this functionality.  

- Doing this will make adding projects in the future easy.  You will only add it in one place - as another object in the projects array. I strongly advise adding this feature to your portfolio as it was missed by the previous reviewer 



- [ ] Please adjust the spacing between elements in your design to look like the Figma design, see screenshot below ⬇️ 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Template 2- Detail](https://user-images.githubusercontent.com/69638013/217255160-2bfbbdbc-1740-40e4-a9de-15592185b6bc.png)  |  ![Screenshot_693](https://user-images.githubusercontent.com/69638013/217254815-68394fcb-4f73-49ab-82b7-a4569f55502d.png)